index.js

```diff
@@ -171,7 +150,7 @@ class TuyaDevice extends EventEmitter {
    *             '1': true,
    *             '2': 'white'
    *          }}).then(() => console.log('device was changed'))
-   * @returns {Promise<Boolean>} - returns `true` if the command succeeded
+   * @returns {Promise<Object>} - returns response from device
```

Comment:
Love this, far more useful return. This may be a breaking change if folks have written their scripts to check for a boolean response. Consider version bump?

Reply:
As long as users have done something like 
```javascript
if (result) {
  console.log('Worked!');
}
```

or 

```javascript
if (!result) {
  console.error('Error!');
}
```

and not

```javascript
if (result == true) {
  console.log('Worked!');
}
```

they should be fine.

It's technically a breaking change, so according to SemVer we should increment.  But I dunno, it's not like we've added any new features.

Reply:
That's the case I'm noting, yes technically semver would specify a version bump, and yes it's a "small" breaking change, so I don't think it would be terrible. On the other hand, version bump doesn't really "cost" anything. Ultimately your call, I'm fine either way.

Reply:
Ok, I'll bump the major version just to keep it consistent.